platformio.ini

```diff
@@ -0,0 +1,70 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+src_dir = main
+
+[env]
+build_flags = 
+    -Iinclude/tls_config
+    -DFRAMEWORK_$PIOFRAMEWORK
```

Comment:
[env] section contains defaults for all platforms; must still be explicitly referenced using ${env. ... }, see below.